.gitignore

```diff
@@ -11,3 +11,4 @@
 !.logs/.gitkeep
 data/reports/**
 data/adversaries/**
+app/parsers/**
```

Comment:
The parsers are considered source code, so they should be tracked in source control, any reason to ignore them?

Reply:
Im guessing @heatonk did that to make sure we didnt upload a parser that is still not public, and then forgot to take it out.

Ill remove it real quick.

Reply:
@mrengstrom - should be good now. Good catch.

Reply:
Thanks, and sounds good 👍